modules/cloud-identity-group/README.md

```diff
@@ -0,0 +1,50 @@
+# Cloud Identity Group Module
+
+This module allows creating a Cloud Identity group and assigning owners, managers and members.
+
+## Usage
+To use this module you must either run terraform as a user that has the Super Admin role in Cloud Identity or [enable domain-wide delegation](https://developers.google.com/admin-sdk/directory/v1/guides/delegation) to the service account used by terraform. If you use a service account, you must also grant that service account the Groups Admin role in Cloud Identity.
```

Comment:
A Cloud Identity custom admin with [groups admin privileges](https://support.google.com/a/answer/1219251#groups) is enough and a better recommendation.
Note that a Service Account **can** be given custom admin privileges. 
https://workspaceupdates.googleblog.com/2020/08/service-accounts-in-google-groups-beta.html

Reply:
Why do we need a custom admin? Isn't that exactly what the predefine "Groups Admin" role provides?